cmd/ci-operator/main.go

```diff
@@ -92,19 +99,20 @@ func (o *options) Run() error {
 	}
 
 	if !o.dry {
-		log.Println("Setting up namespace for testing...")
-		projectGetter, err := versioned.NewForConfig(o.clusterConfig)
-		if err != nil {
-			return fmt.Errorf("could not get project client for cluster config: %v", err)
-		}
-
-		if _, err := projectGetter.ProjectV1().ProjectRequests().Create(&projectapi.ProjectRequest{
-			ObjectMeta: meta.ObjectMeta{
-				Namespace: o.jobSpec.Identifier(),
-				Name:      o.jobSpec.Identifier(),
-			},
-		}); err != nil && ! errors.IsAlreadyExists(err) {
-			return fmt.Errorf("could not set up namespace for test: %v", err)
+		if len(o.namespace) == 0 {
```

Comment:
Shouldn't this do an existence check? Just because I passed something doesn't mean it already exists